region/norfair/east/Kronic Boost Room.json

```diff
@@ -552,6 +552,19 @@
         "Jump straight up to collect the drop, then remorph and hold the Crystal Flash inputs."
       ]
     },
+    {
+      "link": [3, 3],
+      "name": "G-Mode Setup - Get Hit By Viola",
+      "requires": [
+        {"ammo": {"type": "Super", "count": 1}},
+        {"heatFrames": 290}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "flashSuitChecked": true,
+      "note": "Use a Super to knock off the second from the bottom Viola. Be sure not to knock it off when it is on the left side, or it will get stock on the bottom platform."
```

Comment:
```suggestion
      "note": "Use a Super to knock off the second from the bottom Viola. Be sure not to knock it off when it is on the left side, or it will get stuck on the bottom platform."
```